tests/__init__.py

```diff
@@ -815,6 +815,11 @@ def snapshot(ignores=None, tracer=ddtrace.tracer, variants=None, async_mode=True
     """
     ignores = ignores or []
 
+    if include_tracer:
+        tracer = Tracer()
+    else:
+        tracer = ddtrace.tracer
```

Comment:
I preferred the previous API

Reply:
With the previous there was no way to specify a new tracer instance on each invocation of the test.